lib/Runtime/Library/JavascriptLibrary.h

```diff
@@ -1236,6 +1236,9 @@ namespace Js
 #if DBG
         void DumpLibraryByteCode();
 #endif
+
+        Var experimentalVersion; // Wasm.experimentalVersion
```

Comment:
Actually I was wondering if we needed to keep this as a field in Library. Couldn't it just be a member of the Wasm object and that's it?
What I mean is, do we need to keep a reference to the member in JavascriptLibrary?